sh/install.sh

```diff
@@ -169,6 +181,15 @@ if [ -z "$PIP_PATH" ]; then
     install_pip
 fi
 
+if [ $OPERATING_SYSTEM == "openSUSE" ]; then
+    echo "Checking pip version"
+    pip_version=`pip --version`
+    pip_python_version="python 3"
+
+    if [[ $pip_version == *$pip_python_version* ]]; then
```

Comment:
Just so I understand, if the installed pip version is the python 3 variant, we instead install the python 2.7 variant here?

Reply:
Yup, 
If the installed pip version is of python 3 variant then the script will use /usr/bin/python2.7 binary instead of /usr/bin/python to install pip under /usr/lib/python2.7/site-packages.  
Basically when the install script runs "/usr/bin/python2.7 /tmp/get-pip.py" it will upgrade pip 7.x version which is pointing to python 3 to pip 9.0.1 version and points to correct python 2.7 lib directory.